scripts/pdf/render.py

```diff
@@ -89,6 +93,12 @@ def main(loc, colorscheme):
         default="basic",
         help="Color scheme of the PDF",
     )
+    parser.add_argument(
+        "-o",
+        "--output",
+        default=None,
```

Comment:
There's not really a reason to have this default to `None` and then within the function apply a default. Just apply the default here.

```suggestion
        default="tldr-pages.pdf",
```